release-1.6/release_team.md

```diff
@@ -0,0 +1,12 @@
+|  **Role** | **Name** |
+|  ------ | ------ |
+|  Lead | Dan Gillespie |
+|  **Secondary** |  |
+|  Features | Ihor Dvoretskyi |
+|  Release Branch | Anthony Yeh |
+|  **Infra** | Jeff Grafton |
+|  Docs | Devin Donnelly |
+|  Bugs | Caleb Miles |
+|  **Testing** | Maru Newby |
```

Comment:
The bolding was originally to denote which roles are missing - currently only Secondary is unfilled.